plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/index.tsx

```diff
@@ -96,7 +122,11 @@ const FieldComponent: React.FC<Props> = ({
                     {Field && (
                       <ArrowIcon icon="arrowDown" size={16} direction="right" expanded={expanded} />
                     )}
-                    <Title>{field.type === "template" ? field.name : fieldName[field.type]}</Title>
+                    <Title>
+                      {field.type === "template"
+                        ? `${field.name}(${getFieldGroup(field.type)})`
+                        : `${fieldName[field.type]}(${getFieldGroup(field.type)})`}
```

Comment:
Can you build the name in a useMemo so it doesn't have to be calculated each render

Reply:
Done!